### DISCUSSION TITLE
Does Github private support alive?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#23740] [New to GitHub]
<p>Hi folks! More than two weeks ago (1-2 Jun) my main account (<a href="https://github.com/velp" rel="nofollow noopener">https://github.com/velp</a>) was flagged. I’ve already created two tickets to support: <span class="hashtag">#713711</span>, <span class="hashtag">#716314</span> but have not got any answers. Also, I’ve created a topic in community forum <a href="https://github.community/t/my-account-was-flagged-as-spammer/116556" class="inline-onebox">My account was flagged as spammer</a> and only here I got some answer (thx AndreaGriffiths11). All of this period of time I cannot work because my work based on this account and commits from this account. Does Github private support work? Anybody got answers from them?</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>Hi! I’ve had a look and this appears to be resolved.</p>
